storage/mysql/log_storage.go

```diff
@@ -51,8 +51,9 @@ const (
 			VALUES(?,0,?,?,?)`
 	insertSequencedLeafSQL = `INSERT INTO SequencedLeafData(TreeId,LeafIdentityHash,MerkleLeafHash,SequenceNumber)
 			VALUES(?,?,?,?)`
-	selectSequencedLeafCountSQL  = "SELECT COUNT(*) FROM SequencedLeafData WHERE TreeId=?"
-	selectLatestSignedLogRootSQL = `SELECT TreeHeadTimestamp,TreeSize,RootHash,TreeRevision,RootSignature
+	selectSequencedLeafCountSQL   = "SELECT COUNT(*) FROM SequencedLeafData WHERE TreeId=?"
```

Comment:
MySQL/InnoDB doesn't keep row counts. Both "count(*) from SequencedLeafData" and "count(*) from Unsequenced" may take a looong time to run.

What do you need those counts for? Would an estimated\* global count be good enough? Is whatever it is that's planning to call this OK with an eventual 20/30\*\* min delay while the query takes its merry way?

Btw, `LogMetadata.GetUnsequencedCounts` probably needs a huge disclaimer stating that it may take forever and should be avoided.

\* aka, "almost completely inaccurate"
\*\* YMMV